docs/whats-new.asciidoc

```diff
@@ -9,10 +9,24 @@
 coming::[7.10.0]
 
 [discrete]
-[[sec-ml-7.10-changes]]
-=== Twelve new {anomaly-detect} configurations
+[[sec-7.10-term-changes]]
+=== Terminology changes
+
+* *Ingest Manager* has been renamed to *Fleet*.
+* *Configuration* has been renamed to *Policy*.
 
-{elastic-sec} now provides additional {anomaly-jobs} for {auditbeat} and
+[discrete]
+[[enhancements-7.10]]
+=== New features
+* New support for macOS 11.0 (Big Sur).
+* Enhanced user interface for the <<admin-page-ov, Endpoint Administration>> page.
+* Add <<trusted-apps-ov, trusted applications>> to avoid performance or compatibility issues.
+* New <<create-eql-rule, Event Correlation rule type>> based on {ref}/eql.html[EQL (Event Query Language)]. 
+* New <<create-indicator-rule, Indicator Match rule type>> to create alerts for index field values that match threat indices.
+* Free, open detections in the https://github.com/elastic/detection-rules/issues[Detection Rules repo].
```

Comment:
I would link to https://github.com/elastic/detection-rules#detection-rules instead of a list of issues. That way people know how to utilize the detection rules contained in the repo.